packages/code-du-travail-api/src/server/routes/themes/search.elastic.js

```diff
@@ -33,7 +29,11 @@ function getThemeQuery({ slug }) {
     ],
     query: {
       bool: {
-        filter: [{ term: { slug } }, { term: { isPublished: true } }],
+        filter: [
+          { term: { slug: slug.replace(/^\d*-/, "") } },
```

Comment:
is this needed since theme slug doesn't contains number now ?

Reply:
Yep to avoid breaking old routes (but I could also do a redirect with next, might be more appropriate)

Reply:
old route will be handled on /themes routes. getThemeQuery will search theme documents whose don't have `id` in their slug.

Reply:
I'm not sure I understand. FYI old routes => `themes/45-demission`,  new routes => `themes/demission`

Reply:
how does the old gets handled by `themes` routes ? We get a 404 and that's all I thnk